src/documentation/domains/issue/controller/issue.controller.docs.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     TimeTrackingEvent:
+ *       type: object
+ *       properties:
+ *         id:
+ *           type: string
+ *           description: The ID of the event.
+ *         startTime:
+ *           type: string
+ *           format: date-time
+ *           description: The start time of the event.
+ *         endTime:
+ *           type: string
+ *           format: date-time
+ *           description: The end time of the event.
+ *         issueId:
+ *           type: string
+ *           description: The ID of the issue associated with the event.
+ *   parameters:
+ *     IssueIdParam:
+ *       in: path
+ *       name: id
```

Comment:
Should change the class recieved name (IssuePauseParams)

Reply:
Solved.